src/api/routers/config.py

```
@@ -0,0 +1,345 @@
+            "system": platform.system(),
+            "machine": platform.machine(),
+            "processor": platform.processor(),
+        }
+
+        # Try to get package versions
+        try:
+            import fastapi
+
+            env_info["fastapi_version"] = fastapi.__version__
+        except Exception:
+            pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)